### PR TITLE
Adding various fancy goods from Doe's icon set.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Adds bowls, goblets, decanters and vases from the giant pile of icons @PlayerDeer has made recently. Nothing uses them yet but they are available for subtyping/mapping.

image

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Depends on #4592

File: code/modules/crafting/stack_recipes/recipes_planks.dm [hunk 1]
```diff
@@ -78,6 +78,26 @@
 /decl/stack_recipe/planks/bowl
 	result_type            = /obj/item/chems/glass/handmade/bowl
 
+/decl/stack_recipe/planks/fancy
```
Comment (out-of-phaze) [inline code/modules/crafting/stack_recipes/recipes_planks.dm:81]: huh, why are the fancy cups... wooden?

**Review by out-of-phaze — APPROVED** [review]

File: code/modules/crafting/stack_recipes/recipes_planks.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/modules/crafting/stack_recipes/recipes_planks.dm:81]: Because we don't really have metalcrafting yet.

Comment (MistakeNot4892) [reply code/modules/crafting/stack_recipes/recipes_planks.dm:81]: Oh, I remembered, this is so you can make molds of them. Could put them on stone instead possibly.

Comment (out-of-phaze) [reply code/modules/crafting/stack_recipes/recipes_planks.dm:81]: no that makes sense!

**Pull request merged**